packages/flutter/lib/src/gestures/events.dart

```diff
@@ -21,18 +21,66 @@ enum PointerDeviceKind {
   mouse
 }
 
+/// The bit of [PointerEvent.buttons] that coorresponds to the primary mouse button.
```

Comment:
typo: corresponds

Looks like this got copied/pasted into a bunch of other comments too.